app/assets/javascripts/views/story_view.js

```diff
@@ -116,57 +116,13 @@ module.exports = FormView.extend({
   // Triggered whenever a story is dropped to a new position
   sortUpdate: function(ev, ui) {
     // The target element, i.e. the StoryView.el element
-    var target = $(ev.target);
-
-    // Initially, try and get the id's of the previous and / or next stories
-    // by just searching up above and below in the DOM of the column position
-    // the story was dropped on.  The case where the column is empty is
-    // handled below.
-    var previous_story_id = target.prev('.story').data('story-id');
-    var next_story_id = target.next('.story').data('story-id');
-
-    // Set the story state if drop column is chilly_bin or backlog
-    var column = target.parent().attr('id');
-    if (column === 'backlog' || (column === 'in_progress' && this.model.get('state') === 'unscheduled')) {
-      this.model.set({state: 'unstarted'});
-    } else if (column === 'chilly_bin') {
-      this.model.set({state: 'unscheduled'});
-      [previous_story_id, next_story_id] = [next_story_id, previous_story_id];
-    }
-
-    // If both of these are unset, the story has been dropped on an empty
-    // column, which will be either the backlog or the chilly bin as these
-    // are the only columns that can receive drops from other columns.
-    if (_.isUndefined(previous_story_id) && _.isUndefined(next_story_id)) {
-
-      var beforeSearchColumns = this.model.collection.project.columnsBefore('#' + column);
-      var afterSearchColumns  = this.model.collection.project.columnsAfter('#' + column);
+    const target = $(ev.target);
 
-      var previousStory = _.last(this.model.collection.columns(beforeSearchColumns));
-      var nextStory = _.first(this.model.collection.columns(afterSearchColumns));
-
-      if (typeof previousStory !== 'undefined') {
-        previous_story_id = previousStory.id;
-      }
-      if (typeof nextStory !== 'undefined') {
-        next_story_id = nextStory.id;
-      }
-    }
+    const previous_story_id = target.prev('.story').data('story-id');
```

Comment:
Can we change the places that are using `snake_case` in this file to use `camelCase`? It's more common in JS.

Reply:
Done! I'll try to check other files while working in this story sort problem.